src/libraries/Common/src/System/SR.cs

```diff
@@ -7,7 +7,10 @@ namespace System
 {
     internal static partial class SR
     {
-        private static readonly bool s_usingResourceKeys = AppContext.TryGetSwitch("System.Resources.UseSystemResourceKeys", out bool usingResourceKeys) ? usingResourceKeys : false;
+        private static readonly bool s_usingResourceKeys = GetUsingResourceKeysSwitchValue();
+
+        // This method is a target of ILLink substitution.
+        private static bool GetUsingResourceKeysSwitchValue() => AppContext.TryGetSwitch("System.Resources.UseSystemResourceKeys", out bool usingResourceKeys) ? usingResourceKeys : false;
```

Comment:
> I think this could be changed to use `FeatureSwitchDefinition`, and then instead of featuredefault, set the defaults in `Microsoft.NET.ILLink.targets`. Do you see any problems with that?

Much simpler. Surprising nobody thought of this in the issue, we already had the facilities. FeatureSwitchDefinition didn't look necessary so I didn't add it - it wasn't clear to me what it would buy here.